src/sparseml/pytorch/optim/modifier_pruning.py

```diff
@@ -77,117 +79,95 @@ def _log_sparsity(
             )
 
 
-@PyTorchModifierYAML()
-class ConstantPruningModifier(ScheduledModifier):
+class _PruningParamsModifier(ScheduledUpdateModifier):
```

Comment:
would it make sense to add a `finalize` method that checks if the pruning masks are enabled, and if so, calls `apply` one last time and then disables them?

Reply:
looks like you read my mind

Reply:
:)